config/README.md

```diff
@@ -381,6 +381,17 @@ test_extras = """
 """
 ```
 
+If your package uses [`mxdev`](https://pypi.org/project/mxdev/) to handle source checkouts for dependencies, you can set the `use_mxdev` key to ensure `tox` will first run mxdev.
+
+You also will need to manually set the installation of additional packages being pulled by `mxdev` in the `test_deps_additional` key.
+
+```toml
+[tox]
+use_mxdev = true
+constrain_package_deps = false
```

Comment:
🤔 so we need this in order to be able to install development versions of packages?

But isn't `mxdev`'s purpose to generate a `constraints.txt` that would already give us that? I really have to start using it so I don't ask _dumb_ questions 🤓 😅 

i.e. the local `constraints-mxdev.txt` file generated by `mxdev` (during the `tox -e init` phase, if I got it right) should generate a valid constraints.txt file so that I can have all versions pinned while allowing any version for the ones in development mode, or? 🤔

Reply:
According to [tox docs](https://tox.wiki/en/latest/config.html#constrain_package_deps), `constrain_package_deps` set to true will generate a constraints.txt file from the provided options.
In my tests `tox` will bark if we pass `constrain_package_deps = true` and also provide a `-c constraints-mxdev.txt`, that is why I explicitly set to false.

Reply:
Yeah, I saw it the other day also, when trying to configure `plone.app.discussion` to use the unreleased version of `plone.app.robotframework`